schema.graphql

```diff
@@ -29887,31 +29984,6 @@ type ProjectView implements Node {
   groupBy: [Int!] @deprecated(reason: "The `ProjectNext` API is deprecated in favour of the more capable `ProjectV2` API. Follow the ProjectV2 guide at https://github.blog/changelog/2022-06-23-the-new-github-issues-june-23rd-update/, to find a suitable replacement. Removal on 2022-10-01 UTC.")
   id: ID!
 
-  """
-  The view's filtered items.
-  """
-  items(
```

Comment:
See above comment on `ProjectV2View.items`.

**This change is safe to merge.**